pep_sphinx_extensions/pep_zero_generator/pep_index_generator.py

```diff
@@ -30,9 +32,25 @@
     from sphinx.environment import BuildEnvironment
 
 
-def create_pep_zero(_: Sphinx, env: BuildEnvironment, docnames: list[str]) -> None:
-    # Sphinx app object is unneeded by this function
+def create_pep_json(peps: list[parser.PEP], out_dir: str) -> None:
+    pep_list = [
+        {
+            "number": pep.number,
+            "title": pep.title,
+            "authors": ", ".join([pep.authors.nick for pep.authors in pep.authors]),
```

Comment:
```suggestion
            "authors": ", ".join(pep.authors.nick for pep.authors in pep.authors),
```
Don't create a temp list